capi/src/symbolize.rs

```diff
@@ -566,10 +566,10 @@ pub unsafe extern "C" fn blaze_symbolize_kernel_virt_addrs(
 }
 
 
-/// Symbolize file addresses in an ELF file.
+/// Symbolize addresses representing virtual offsets in an ELF file.
```

Comment:
so it's a bit confusing to read "addresses representing virtual offsets". I see that we literally have "addrs" parameter name, but maybe here in the description it would be nicer to just say something like  "Symbolize virtual offsets" and name parameter as "virt_offsets" (and `virt_offset_cnt` then, of course), to make it a bit less confusing? Could be just "offsets", but given we anticipate file offset variant of this API, might be a bit more verbose here. Don't know, just thoughts, don't feel strongly.

Reply:
I agree. Will adjusts as per your suggestion.